setup.py

```diff
@@ -7,14 +7,14 @@
 
 extras_require = {
     'test': [
-        "pytest==3.3.2",
+        "pytest==5.4.1",
         "pytest-xdist",
-        "tox>=2.9.1,<3",
+        "tox==3.14.6",
```

Comment:
`tox` has respected semver as far as I remember (unlike pytest/flake8), but no real objection to pinning it, either. (Objections to pinning are usually about doing quick security releases, but that's not too important for tox, which isn't released with the deployed package)